### ISSUE TITLE
aws_elastic_lb: No method named 'elb' with cookbook version 3.3

### ISSUE THREAD

**Issue by christinedraper** [chef-cookbooks/aws#236]
This works with cookbook version 2.9.3:
```
    include_recipe 'aws'
    aws_elastic_lb("mylb") do
      action :register
    end
```
and fails with 3.3.1:
```
Generated at 2016-04-13 15:25:58 +0000
NameError: aws_elastic_lb[oeapp-test-testvpc-lb] (oe_app::register line 19) had an error: NameError: No resource, method, or local variable named `elb' for `LWRP provider aws_elastic_lb from cookbook aws ""'
/opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-12.8.1/lib/chef/dsl/recipe.rb:104:in `rescue in method_missing'
/opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-12.8.1/lib/chef/dsl/recipe.rb:99:in `method_missing'
/var/chef/cache/cookbooks/aws/providers/elastic_lb.rb:7:in `block (2 levels) in class_from_file'
/opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-12.8.1/lib/chef/mixin/why_run.rb:52:in `add_action'
/opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-12.8.1/lib/chef/provider.rb:175:in `converge_by'
/var/chef/cache/cookbooks/aws/providers/elastic_lb.rb:6:in `block in class_from_file'
(eval):2:in `block in action_register'
/opt/chef/embedded/lib/ruby/gems/2.1.0/gems/chef-12.8.1/lib/chef/provider.rb:360:in `instance_eval'
<snip>
```
It seems it is not installing the aws sdk before trying to reference the elb client.

**Comment by sneakycactus** [comment]
It appeared to be a missing reference to the Elb module when I resolved this. I submitted the following:

https://github.com/chef-cookbooks/aws/pull/235

**Comment by tas50** [comment]
Fixed 3.3.2

**Comment by christinedraper** [comment]
wow. thanks for the fast turnaround
